README.md

```diff
@@ -1,56 +1,63 @@
-# Cosmos SDK
-
-![banner](docs/graphics/cosmos-sdk-image.png)
-
-[![version](https://img.shields.io/github/tag/cosmos/cosmos-sdk.svg)](https://github.com/cosmos/cosmos-sdk/releases/latest)
-[![API Reference](https://godoc.org/github.com/cosmos/cosmos-sdk?status.svg
-)](https://godoc.org/github.com/cosmos/cosmos-sdk)
-[![Rocket.Chat](https://demo.rocket.chat/images/join-chat.svg)](https://cosmos.rocket.chat/)
-[![license](https://img.shields.io/github/license/cosmos/cosmos-sdk.svg)](https://github.com/cosmos/cosmos-sdk/blob/master/LICENSE)
-[![LoC](https://tokei.rs/b1/github/cosmos/cosmos-sdk)](https://github.com/cosmos/cosmos-sdk)
-[![Go Report Card](https://goreportcard.com/badge/github.com/cosmos/cosmos-sdk)](https://goreportcard.com/report/github.com/cosmos/cosmos-sdk)
-
-Branch    | Tests | Coverage
-----------|-------|---------
-develop   | [![CircleCI](https://circleci.com/gh/cosmos/cosmos-sdk/tree/develop.svg?style=shield)](https://circleci.com/gh/cosmos/cosmos-sdk/tree/develop) | [![codecov](https://codecov.io/gh/cosmos/cosmos-sdk/branch/develop/graph/badge.svg)](https://codecov.io/gh/cosmos/cosmos-sdk)
-master    | [![CircleCI](https://circleci.com/gh/cosmos/cosmos-sdk/tree/master.svg?style=shield)](https://circleci.com/gh/cosmos/cosmos-sdk/tree/master) | [![codecov](https://codecov.io/gh/cosmos/cosmos-sdk/branch/master/graph/badge.svg)](https://codecov.io/gh/cosmos/cosmos-sdk)
-
-
-WARNING: the libraries are still undergoing breaking changes as we get better ideas and start building out the Apps.
-
-**Note: Requires Go 1.9+**
-
-The Cosmos SDK is a platform for building multi-asset Proof-of-Stake
-cryptocurrencies, like the [Cosmos Hub](https://cosmos.network). It is both a
-library for building applications and a set of tools for securely interacting
-with them.
-
-The goals of the SDK are to abstract away the complexities of building a
-Tendermint ABCI application in Golang and to provide a framework for building
-interoperable blockchain applications in the Cosmos Network.
-
-It is inspired by capabilities-based security, and informed by years of
-wrestling with blockchain state machines. 
```

Comment:
I liked this paragraph as a concise overview of the whole thing. Is there someway we can still include it? Or you think not worth it given the new more complete descriptions ?